.github/workflows/create_conda_envs.yml

```diff
@@ -0,0 +1,298 @@
+name: Create Conda Environments in Docker
+
+on:
+  workflow_dispatch:
+    inputs:
+      subset:
+        description: 'Comma-separated list of environments to skip. Leave blank to build all.'
```

Comment:
@georgemccabe do you find this logic useful, list the envs to "skip" rather than the envs to "build". Personally, I would expect that an empty list would mean build all possible envs, while if the list is not empty, only build the ones listed. That's how the filtering logic for many MET config options work.

But if this logic is useful, it also works.

Reply:
The logic actually builds the list you supply, not skips them. I had originally thought I'd want to supply a list to skip when I wrote the description, but I forgot to update it when I changed the logic. Thanks for noticing that.